build.sbt

```diff
@@ -330,6 +331,41 @@ lazy val tests: ProjectMatrix = (projectMatrix in file("tests"))
   )
   .dependsOn(core, circeJson, cats)
 
+
+val akkaHttpOnly = taskKey[Unit]("akka-http-only")
+val akkaHttpTapir = taskKey[Unit]("akka-http-tapir")
+lazy val perfTests: ProjectMatrix = (projectMatrix in file("perf-tests"))
+  .enablePlugins(GatlingPlugin)
+  .settings(commonJvmSettings)
+  .settings(
+    name := "tapir-perf-tests",
+    libraryDependencies ++= Seq(
+      "io.gatling.highcharts" % "gatling-charts-highcharts" % "3.6.1" % "test",
+      "io.gatling"            % "gatling-test-framework"    % "3.6.1" % "test",
+      "com.typesafe.akka" % "akka-actor-typed_2.13" % "2.6.18",
+      "com.typesafe.akka" % "akka-stream-typed_2.13" % "2.6.18",
```

Comment:
you can use `%%` to avoid the `_2.13` suffix, and `Versions.akkaStreams` instead of `2.6.18` no to repeat the version number